packages/nx/bin/nx.ts

```diff
@@ -60,21 +61,37 @@ function main() {
     try {
       localNx = resolveNx(workspace);
     } catch {
-      // If we can't resolve a local copy of Nx, we must be global.
-      warnIfUsingOutdatedGlobalInstall();
-      output.error({
-        title: `Could not find Nx modules in this workspace.`,
-        bodyLines: [`Have you run ${chalk.bold.white(`npm/yarn install`)}?`],
-      });
-      process.exit(1);
+      localNx = null;
+    }
+
+    const isLocalInstall = localNx === resolveNx(null);
+    const LOCAL_NX_VERSION: string | null = localNx
+      ? getLocalNxVersion(workspace)
+      : null;
+    const GLOBAL_NX_VERSION: string | null = isLocalInstall
+      ? null
+      : require('../package.json').version;
+
+    process.env.GLOBAL_NX_VERSION ??= GLOBAL_NX_VERSION;
+    process.env.LOCAL_NX_VERSION ??= LOCAL_NX_VERSION;
+
+    if (process.argv[2] === '--version') {
+      console.log(stripIndents`Nx Version:
+      - Local: v${LOCAL_NX_VERSION ?? 'Not found'}
+      - Global: v${GLOBAL_NX_VERSION ?? 'Not found'}`);
```

Comment:
If we can't find it, let's just not show it? It means they're using the local version right?

Reply:
Yeah - we can hide it. I'd like to keep the logic for local Nx though, as that shows that the workspace couldn't be found

Reply:
Yes, I agree.